### PR TITLE
Update/function param

### PR DESCRIPTION
**Pull request by inwookie**
# Description (개요) 

ThanoSQL Registry의 Function 함수들이 수정됨에 따라 docs도 필요한 수정 했습니다. 

## Keep in Mind

If you are creating a new statement, your first commit should always be a copy of the statement(model, ThanoSQL, function) template. This will allow reviewers to see what has been changed from the template, making the review process much more efficient.

## Type of change (작업사항)

Please delete options that are not relevant and check out the type of change.

- [x] Function statement fix 

Following templates can be found from the [Tech-wiki Docs](https://github.com/smartmind-team/tech-wiki/tree/main/project/thanosql/docs). 

## Writing Convention

Make sure to check out the [Writing Convention](https://github.com/smartmind-team/tech-wiki/blob/main/project/thanosql/docs/writing_convention.md). 

## Review Process

Review should be conducted by at least two people.

### CODE REVIEW

File: docs/ko/how-to_guides/ThanoSQL_query/FUNCTION_SYNTAX.md [hunk 1]
```diff
@@ -34,9 +34,14 @@ OPTIONS (
...
 !!! note "'__FUNCTION__ 문'을 사용할 수 있는 ThanoSQL의 모듈과 함수"
     - compute
-        - groupby_mean
-        - similiarities
+        - groupby_emb_mean
+        - similarity
     - preprocess
-        - video_to_df
\ No newline at end of file
+        - video_to_df
```
Comment (yeemh) [inline docs/ko/how-to_guides/ThanoSQL_query/FUNCTION_SYNTAX.md:47]:
```suggestion
        - video_to_df
```

**Review by yeemh — APPROVED** [review]
잘 못 들어간 스페이스만 지워주세요.
변수명 같은 경우는 다 얘기가 된거죠?

File: docs/ko/how-to_guides/ThanoSQL_query/FUNCTION_SYNTAX.md (hunk 1 above)
Comment (inwookie) [reply docs/ko/how-to_guides/ThanoSQL_query/FUNCTION_SYNTAX.md:47]: 예스!!!!

Comment (yeemh) [reply docs/ko/how-to_guides/ThanoSQL_query/FUNCTION_SYNTAX.md:47]: 👍

**Pull request merged**